app/main.py

```diff
@@ -2,4 +2,56 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if not coords else coords
```

Comment:
Try to avoid using `not` by reverting the if condition in the future
```suggestion
        self.coords = coords if coords else [0, 0]
```